feat(changelog): object-storage-fixed-slowness-issues-are-fixed 2023-03-09 (#1405)
scaleway/docs-content: changelog/march2023/2023-03-09-object-storage-fixed-slowness-issues-are-fixed.mdx

---

> title: Slowness issues are fixed in Paris Region

Reviewer: ```suggestion
title: Slowness issues fixed in Paris Region
```

status: fixed
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-03-09
category: storage
product: object-storage
---

> Some customers had experienced slowness issues in fr-par over the last few weeks. The problem is fixed and latency has been divided by 3 starting Feb, 24th around 4:30PM.

Author: ```suggestion
Some customers experienced slowness issues in fr-par over the last few weeks. The problem is now resolved and latency has been divided by 3 starting Feb, 24th around 4:30 PM.
```

> Some customers experienced slowness issues in fr-par over the last few weeks. The problem is now resolved and latency has been divided by 3 starting Feb, 24th around 4:30 PM.

Reviewer: ```suggestion
Some customers experienced slowness issues in `fr-par` over the last few weeks. The problem is now resolved and latency has been divided by 3 starting Feb, 24th around 4:30 PM.
```
